ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/Event.java

```diff
@@ -116,6 +117,17 @@ public EventBuilder<T, R, S> showSummary() {
       return this;
     }
 
+
+    public EventBuilder<T, R, S> publish(boolean publish) {
```

Comment:
Is there any way we can make it clearer what this flag is? What does publish do?

Reply:
Great, should we call the field and method `publishToCamunda`?

Reply:
I am afraid we cannot, we need to use Publish as the name in order that the event gets picked up to be published by the wa-case-event-handler. I checked with other teams likes SCSS, IA & ET , they are all using the same flag name.

Reply:
That's not true. The field and method can be publishToCamunda. The JSON output will be Publish

Reply:
Oh yes, you meant just the variable name. Yes that can be done, I'll do the change.